docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "2"
+services:
+  db:
+    image: "postgres:9.5"
```

Comment:
Right. To be precise, there are two separate containers here, `db` and `web`, and any container listed here will by default be able to talk to any other container listed here. In order to talk to outside containers or things outside of Docker, then you need to expose the port.